pkg/signal/signal.go

```diff
@@ -11,6 +11,11 @@ import (
 	"syscall"
 )
 
+const (
+	// DefaultStopTimeout is the timeout (in seconds) for the syscall signal used to stop a container.
+	DefaultStopTimeout = 10
```

Comment:
nit, this doesn't seem like the correct place.
Probably should go in either the daemon package or the container package.